src/test/kotlin/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,55 @@
+package calculator
+
+import io.kotest.assertions.throwables.shouldThrow
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.data.forAll
+import io.kotest.data.row
+import io.kotest.matchers.shouldBe
+
+class CalculatorTest : StringSpec({
+
+    "덧셈" {
+        Calculator().calculate("1 + 2") shouldBe 3
+    }
+
+    "뺄셈" {
+        Calculator().calculate("1 - 2") shouldBe -1
+    }
+
+    "곱셈" {
+        Calculator().calculate("1 * 2") shouldBe 2
+    }
+
+    "나눗셈" {
+        Calculator().calculate("1 / 2") shouldBe 0
+    }
+
+    "입력값이 공백이면 예외 발생" {
+        forAll(
+            row(""),
+            row(" "),
+        ) { input ->
+            val exception = shouldThrow<IllegalArgumentException> {
+                Calculator().calculate(input)
+            }
```

Comment:
람다 파라미터가 1개뿐이라면 **파라미터명을 직접 명시할 필요 없이** `it`으로 사용할 수 있어요! 😁

```suggestion
        ) {
            val exception = shouldThrow<IllegalArgumentException> {
                Calculator().calculate(it)
            }
```

Reply:
이 부분은 가독성 측면에서 호불호가 좀 생기는 것 같은데, 
선생님은 혹시 항상 it 을 사용하시는 것을 선호하시나요?
저는 상황별로 반반 섞어 사용하는 것 같습니다. 파라미터 변수로 명시하면 1줄 더 생겨서 길어보이지만 읽기는 편하더라구요.
일단 여기서는 단순해서 괜찮을 것 같으니 말씀하신 의견에 따라 it 으로 고쳐볼게요 😊

Reply:
저 같은 경우에는 it을 많이 사용하고있어요. 😊 
가독성 측면 때문에 걱정했던 시기도있으나... 자주 쓰다보니 코드를 읽는데도 그렇게 버벅임 없이 잘 읽혀지더라구요. 🤔 
물론 이 경우도 다른 변수명을 잘 지은 경우에 잘 읽히는거지 앞에서부터 변수명의 의미가 불명확하다면 `it`은 더큰 혼란을 가져오더라구요. 😅

예를들어,  상품 리스트를 **products**로 지정했다면. `products.forEach { /* it */ }` 이라고 뒀다면  it이 products의 한 상품인 `product`를 의미하겠구나하고 빠르게 추측이되는데,  **productBundle**(bundle은 묶음을 의미)로 지정했다면 `products.forEach { /* it */ }`에서 it의 의미가 빨리 파악되지 않을 것 같아요.

추가적으로 여러개의 람다식을 연달아 사용하는 경우에는 `it`을 지양하는 편이고(아마 IntelliJ가 it이 무엇을 지칭하는지 몰라서 그렇게 못쓴쓰지말라고했던걸로 기억해요 😁),

Reply:
답변이 유용하고 도움이 되네요 감사합니다. 😄